.github/workflows/doc-build.yml

```
@@ -95,10 +95,11 @@ jobs:
         # If matches a tag like refs/tags/v1.12.0-rc3 or
         # refs/tags/v1.12.0 convert to 1.12
         GITHUB_REF=${{ github.ref }}
+        export GITHUB_REF=${{ github.ref }}
```

The first line is redundant
```suggestion
        export GITHUB_REF=${{ github.ref }}
```